dev-tools/mage/gotool/go.go

```diff
@@ -58,7 +58,9 @@ var Test goTest = runGoTest
 
 // GetModuleName returns the name of the module.
 func GetModuleName() (string, error) {
-	lines, err := getLines(callGo(nil, "list", "-m"))
+	// `go list -buildvcs=false -f '{{.Module}}'` produces the same output as `go list -m`
+	// and it works with go workspace
+	lines, err := getLines(callGo(nil, "list", "-buildvcs=false", "-f", "{{.Module}}"))
```

Comment:
Do we need `-buildvcs=false` here?  AFAICT it's not an option supported by `go list`.  It looks like that option is only supported by `go build`.  Also, running just `go list -f {{.Module}}` seems to work.